app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
+    people = {}
     pass
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    for person in people:
+        Person(person["name"], person["age"])
```

Comment:
Create a list in which, during the first iteration, you add Person objects without the spouse field, and in the second iteration, add them.